pytensor/tensor/sort.py

```diff
@@ -1,25 +1,32 @@
+from typing import Literal
+
 import numpy as np
 
 from pytensor.gradient import grad_undefined
-from pytensor.graph.basic import Apply, Constant
+from pytensor.graph.basic import Apply
 from pytensor.graph.op import Op
 from pytensor.misc.safe_asarray import _asarray
 from pytensor.tensor.basic import arange, as_tensor_variable, switch
-from pytensor.tensor.math import eq, ge, mul
+from pytensor.tensor.math import eq, ge
 from pytensor.tensor.type import TensorType
 
 
-def _variable_is_none(var):
-    return isinstance(var, Constant) and var.data is None
+KIND_VALUES = ["quicksort", "mergesort", "heapsort", "stable"]
+KIND = Literal["quicksort", "mergesort", "heapsort", "stable"]
 
 
-def _check_tensor_is_scalar(var):
-    """
-    Checks if a tensor variable is scalar, raise ValueError otherwise
-    """
-    msg = "%(var)s is expected to be 0d tensor, got %(ndim)d"
-    if var.ndim != 0:
-        raise ValueError(msg % (var, var.ndim))
+def _parse_sort_args(kind: KIND | None, order, stable: bool | None) -> KIND:
+    if order is not None:
+        raise ValueError("order is not applicable to PyTensor graphs")
```

Comment:
```suggestion
        raise ValueError("The "order" argument not used by PyTensor and cannot be set. It is included for consistency with numpy only")
```

Bit more obvious what's going on

Reply:
It is included only so we can raise this informative error :D